inspire/dojson/schemas/hep-0.0.1.json

```diff
@@ -0,0 +1,646 @@
+                },
+                "method": {
+                    "enum": []
+                },
+                "source": {
+                    "enum": []
+                },
+                "submission_id": {
+                    "type": "string"
+                }
+            },
+            "required": ["submission_id"],
+            "type": "object"
+        },
+        "authors": {
+            "description": "List with all the authors",
+            "items": {
+                "properties": {
+                    "affiliations": {
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "value": {
+                                    "type": "string"
+                                },
+                                "recid": {
+                                    "type": "integer"
+                                },
+                                "curated_relation": {
+                                    "type": "bool"
+                                }
+                            }
+                        },
+                        "type": "array"
+                    },
+                    "email": {
+                        "format": "email",
+                        "type": "string"
+                    },
+                    "inspire_bai": {
+                        "description": "INSPIRE BAI",
+                        "type": "string",
+                        "pattern": "(\\w+\\.)+\\d+"
+                    },
+                    "inspire_id": {
+                        "description": "Legacy INSPIRE ID",
+                        "type": "string",
+                        "pattern": "INSPIRE-\\d+"
+                    },
+                    "name": {
+                        "type": "string"
+                    },
+                    "orcid": {
+                        "description": "ORCID Id",
+                        "format": "orcid",
+                        "type": "string",
+                        "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{4}"
+                    },
+                    "other_ids": {
+                        "type": "array",
+                        "items": {
+                            "properties": {
+                                "source": {
+                                    "type": "string"
+                                },
+                                "value": {
+                                    "type": "string"
+                                }
+                            },
+                            "type": "object"
+                        }
+                    },
+                    "role": {
+                        "type": "string"
+                    },
+                    "recid": {
+                        "type": "integer"
+                    },
+                    "curated_curated": {
+                        "type": "bool"
+                    }
```

Comment:
... and here, there is an example of how to deal with relations. Shall we always go for `recid`? Any better pattern? @tiborsimko, @egabancho, @jirikuncar?